clusters/development/overlay/third-party/snyk-monitor/helmRelease.yaml

```diff
@@ -7,3 +7,10 @@ spec:
   chart:
     spec:
       version: 1.80.0 # https://artifacthub.io/packages/helm/snyk/snyk-monitor
+  values:
+    requests:
+      cpu: "1"
+      memory: "1Gi"
+    limits:
+      cpu: "2"
+      memory: "2Gi"
```

Comment:
Optionally - suggest to keep limits and requests order similar for different configurations - harder to make mistakes. E.g. first `limits`, then `requests`. Related to all changes below.